README.adoc

```diff
@@ -292,17 +364,17 @@ whenever a code change introduces a defect. JUnit and the JAX-RS Client API prov
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `FaultToleranceTest` class.#
-`src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java`
+#Create the `FaultToleranceIT` class.#
+`src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceIT.java`
 ----
 
-The [hotspot=21]`@Before` and [hotspot=27]`@After` annotations indicate that this method executes before or after the other test case.
+The [hotspot=21]`@BeforeEach` and [hotspot=27]`@AfterEach` annotations indicate that this method executes before or after the other test case.
 These methods are generally used to perform any setup and teardown tasks. In this case,
 the setup method creates a JAX-RS client, which makes HTTP requests to the `inventory` service. This
 client must also be registered with a JSON-P provider to process JSON resources.
 The teardown method simply destroys this client instance as well as the HTTP responses.
 
-The [hotspot=34-53]`testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
+The [hotspot=34-54]`testFallbackForGet()` test case sends a request to the `inventory` service to get the systems properties for a
 host name before and after the `system` service becomes unavailable. Then, it asserts outputs from the two requests
```

Comment:
Change
"host name" to "hostname"
(Also in the sentence, "Point your browser to the http://localhost:9080/inventory/systems/localhost[http://localhost:9080/inventory/systems/localhost^] URL, which accesses the `inventory` service with a localhost **host name**.")

Reply:
Updated